Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -45,15 +52,22 @@ protected class HandPanData
         [SerializeField]
         private bool lockVertical = false;
         [SerializeField]
-        private bool wrapTexture = false;
+        [Tooltip("If this is checked, Max Pan Horizontal and Max Pan Vertical are ignored.")]
+        private bool unlimitedPan = true;
+        [SerializeField]
+        [Range(1.0f, 20.0f)]
+        private float maxPanHorizontal = 2;
         [SerializeField]
-        private bool velocityActive = false;
+        [Range(1.0f, 20.0f)]
+        private float maxPanVertical = 2;
         [SerializeField]
         [Range(0.0f, 0.99f)]
-        private float velocityDampingX = 0.9f;
+        [Tooltip("a value of 0 results in panning coming to a complete stop when released.")]
+        private float momentumHorizontal = 0.9f;
         [SerializeField]
+        [Tooltip("a value of 0 results in panning coming to a complete stop when released.")]
         [Range(0.0f, 0.99f)]
-        private float velocityDampingY = 0.9f;
+        private float momentumVertical = 0.9f;
```

Comment:
I think we may need to use the formally serialized as attribute so we don't break existing serialized assets:

https://blogs.unity3d.com/2015/02/03/renaming-serialized-fields/

Reply:
this is a new Script-- HandInteractionPanZoom which adds the zoom functionality without altering the pan-only script HandInteractionPan. So FormerlySerializedAs wouldn't be appropriate since it isn't changing a variable name in an existing script.

Reply:
Unless actually wait these are totally different concepts, in which case I guess this comment is not relevant

Reply:
I think that I just mentioned this automatically because of the line deltas + the fact that the ranges and default values are the same